sources/packages/backend/apps/api/src/route-controllers/student/student.controller.service.ts

```diff
@@ -113,14 +114,36 @@ export class StudentControllerService {
    * @param studentId student id to retrieve the data.
    * @returns student profile details.
    */
-  async getStudentProfile(studentId: number): Promise<StudentProfileAPIOutDTO> {
+  async getStudentProfile(studentId: number): Promise<StudentProfileAPIOutDTO>;
+  /**
+   * Get the student information that represents the profile.
+   * @param studentId student id to retrieve the data.
+   * @param options options:
+   * - `withSensitiveData` boolean option to return sensitive data such as SIN.
+   * @returns student profile details.
+   */
+  async getStudentProfile(
+    studentId: number,
+    options: { withSensitiveData: boolean },
```

Comment:
Minor that we missed in some PRs, if line 127 is not declared as `options: { withSensitiveData: true },` the method consumer can provide it as false which would lead to an incorrect return type.